concordium-sdk/src/main/java/com/concordium/sdk/responses/modulelist/ModuleRef.java

```diff
@@ -21,7 +21,7 @@ public static ModuleRef from(String hexHash) {
         return new ModuleRef(hexHash);
     }
 
-    public static Optional<ImmutableList<ModuleRef>> fromJsonArray(ConcordiumP2PRpc.JsonResponse res) {
+    public static Optional<ImmutableList<ModuleRef>> fromHashJsonArray(ConcordiumP2PRpc.JsonResponse res) {
```

Comment:
This needs to be named `fromHashJsonArray` because the parent `Hash` class also has a method by the name `fromJsonArray` and returns an `ImmutableList<Hash>`. 
@EmilLa1 - please do advise if the name doesn't fit the standards.